feature/google/src/main/java/com/revenuecat/purchases/google/GooglePurchaseOption.kt

```diff
@@ -12,8 +16,5 @@ data class GooglePurchaseOption(
     override val pricingPhases: List<PricingPhase>,
     override val tags: List<String>,
 
-    /**
-     * Token used to purchase
-     */
-    val token: String
+    override val purchasingData: @RawValue PurchasingData
```

Comment:
Hmm right now `GooglePurchaseOption` is not adding anything new to the `PurchaseOption` interface so I'm wondering if we should make `PurchaseOption` the data class and remove this. It might depend on whether we need to add any extra fields in Amazon though.

Reply:
Huh... that _does_  make sense to maybe make things a little cleaner feeling 🤔  But also... this doesn't feel terribly code smelly, IMO 😇 

I would almost feel safer with this approach until we hook Amazon back up fully 🤷‍♂️  Thoughts?

Reply:
maybe we can add a TODO so we don't forget to reconsider?

Reply:
Yeah, I think we can add a `TODO` and reconsider later as @beylmk said.